oidc-controller/api/core/logger_util.py

```diff
@@ -20,3 +21,11 @@ def wrapper(*args, **kwargs):
         return ret_val
 
     return wrapper
+
+
+def extract_error_msg_from_traceback_exc(format_exc) -> str:
+    try:
+        return format_exc.splitlines()[-1].split(": ")[1:][0]
```

Comment:
I'm not completely sure how resilient this will be. It gets the last line of the exception and then gets the message from after the colon.

Reply:
I am not a big fan of having ad-hoc text handling to parse this. Did you look at - potentially - leveraging something like https://docs.python.org/3/library/traceback.html ?

Reply:
Ya. I'm not a fan of it either. I was trying different methods from this library and this was the best I had come up with. 

There is a library logging to the console with out the apps logger with logs like `Unknown client_id 'keycloak'`. You can see it if you don't push the initial configs. I'm not really sure why this is happening. 

I was thinking about looking into that library and see how it's parsing the exception and why it's logging to console.

Reply:
That would be https://github.com/IdentityPython/pyop when checking whether an OIDC client is registered with VC-AuthN. Could be a good idea, but they might just be catching a specific error/exception.

Reply:
Ya that wasn't helpful. I changed it to be a bit cleaner but it's still doing some text parsing. 

Another option is just not returning any details for 500 responses and developers would need to look in the logs, but having some information like this could be useful to end users. Then if there was a 500 error we could see the stack trace in the logs. Catch the exception correctly and raise an accurate HTTPException with details.

Reply:
I think this is the way to go. Making the changes.